// @myparcel/constants/dist/index.js
// src/types/carrier.ts
var CarrierId = /* @__PURE__ */ ((CarrierId3) => {
  CarrierId3[CarrierId3["PostNl"] = 1] = "PostNl";
  CarrierId3[CarrierId3["Bpost"] = 2] = "Bpost";
  CarrierId3[CarrierId3["CheapCargo"] = 3] = "CheapCargo";
  CarrierId3[CarrierId3["Dpd"] = 4] = "Dpd";
  CarrierId3[CarrierId3["Instabox"] = 5] = "Instabox";
  CarrierId3[CarrierId3["Dhl"] = 6] = "Dhl";
  CarrierId3[CarrierId3["Bol"] = 7] = "Bol";
  CarrierId3[CarrierId3["BolCom"] = 7] = "BolCom";
  CarrierId3[CarrierId3["Ups"] = 8] = "Ups";
  CarrierId3[CarrierId3["DhlForYou"] = 9] = "DhlForYou";
  CarrierId3[CarrierId3["DhlParcelConnect"] = 10] = "DhlParcelConnect";
  CarrierId3[CarrierId3["DhlEuroPlus"] = 11] = "DhlEuroPlus";
  CarrierId3[CarrierId3["UpsStandard"] = 12] = "UpsStandard";
  CarrierId3[CarrierId3["UpsExpressSaver"] = 13] = "UpsExpressSaver";
  CarrierId3[CarrierId3["Gls"] = 14] = "Gls";
  return CarrierId3;
})(CarrierId || {});
var CarrierName = /* @__PURE__ */ ((CarrierName3) => {
  CarrierName3["PostNl"] = "postnl";
  CarrierName3["Bpost"] = "bpost";
  CarrierName3["CheapCargo"] = "cheapcargo";
  CarrierName3["Dpd"] = "dpd";
  CarrierName3["Instabox"] = "instabox";
  CarrierName3["Dhl"] = "dhl";
  CarrierName3["Bol"] = "bol.com";
  CarrierName3["BolCom"] = "bol.com";
  CarrierName3["Ups"] = "ups";
  CarrierName3["DhlForYou"] = "dhlforyou";
  CarrierName3["DhlParcelConnect"] = "dhlparcelconnect";
  CarrierName3["DhlEuroPlus"] = "dhleuroplus";
  CarrierName3["UpsStandard"] = "upsstandard";
  CarrierName3["UpsExpressSaver"] = "upsexpresssaver";
  CarrierName3["Gls"] = "gls";
  return CarrierName3;
})(CarrierName || {});
var CARRIER_NAMES_TO_IDS = {
  ["postnl" /* PostNl */]: 1 /* PostNl */,
  ["bpost" /* Bpost */]: 2 /* Bpost */,
  ["cheapcargo" /* CheapCargo */]: 3 /* CheapCargo */,
  ["dpd" /* Dpd */]: 4 /* Dpd */,
  ["instabox" /* Instabox */]: 5 /* Instabox */,
  ["dhl" /* Dhl */]: 6 /* Dhl */,
  ["bol.com" /* Bol */]: 7 /* Bol */,
  ["ups" /* Ups */]: 8 /* Ups */,
  ["dhlforyou" /* DhlForYou */]: 9 /* DhlForYou */,
  ["dhlparcelconnect" /* DhlParcelConnect */]: 10 /* DhlParcelConnect */,
  ["dhleuroplus" /* DhlEuroPlus */]: 11 /* DhlEuroPlus */,
  ["upsstandard" /* UpsStandard */]: 12 /* UpsStandard */,
  ["upsexpresssaver" /* UpsExpressSaver */]: 13 /* UpsExpressSaver */,
  ["gls" /* Gls */]: 14 /* Gls */
};
var CARRIER_IDS_TO_NAMES = {
  [1 /* PostNl */]: "postnl" /* PostNl */,
  [2 /* Bpost */]: "bpost" /* Bpost */,
  [3 /* CheapCargo */]: "cheapcargo" /* CheapCargo */,
  [4 /* Dpd */]: "dpd" /* Dpd */,
  [5 /* Instabox */]: "instabox" /* Instabox */,
  [6 /* Dhl */]: "dhl" /* Dhl */,
  [7 /* Bol */]: "bol.com" /* Bol */,
  [8 /* Ups */]: "ups" /* Ups */,
  [9 /* DhlForYou */]: "dhlforyou" /* DhlForYou */,
  [10 /* DhlParcelConnect */]: "dhlparcelconnect" /* DhlParcelConnect */,
  [11 /* DhlEuroPlus */]: "dhleuroplus" /* DhlEuroPlus */,
  [12 /* UpsStandard */]: "upsstandard" /* UpsStandard */,
  [13 /* UpsExpressSaver */]: "upsexpresssaver" /* UpsExpressSaver */,
  [14 /* Gls */]: "gls" /* Gls */
};

// src/types/customsDeclarationContents.ts
var CustomsDeclarationContents = /* @__PURE__ */ ((CustomsDeclarationContents2) => {
  CustomsDeclarationContents2[CustomsDeclarationContents2["CommercialGoods"] = 1] = "CommercialGoods";
  CustomsDeclarationContents2[CustomsDeclarationContents2["CommercialSamples"] = 2] = "CommercialSamples";
  CustomsDeclarationContents2[CustomsDeclarationContents2["Documents"] = 3] = "Documents";
  CustomsDeclarationContents2[CustomsDeclarationContents2["Gifts"] = 4] = "Gifts";
  CustomsDeclarationContents2[CustomsDeclarationContents2["ReturnShipment"] = 5] = "ReturnShipment";
  return CustomsDeclarationContents2;
})(CustomsDeclarationContents || {});

// src/types/deliveryType.ts
var DeliveryTypeId = /* @__PURE__ */ ((DeliveryTypeId3) => {
  DeliveryTypeId3[DeliveryTypeId3["Morning"] = 1] = "Morning";
  DeliveryTypeId3[DeliveryTypeId3["Standard"] = 2] = "Standard";
  DeliveryTypeId3[DeliveryTypeId3["Evening"] = 3] = "Evening";
  DeliveryTypeId3[DeliveryTypeId3["Pickup"] = 4] = "Pickup";
  DeliveryTypeId3[DeliveryTypeId3["Express"] = 7] = "Express";
  return DeliveryTypeId3;
})(DeliveryTypeId || {});
var DeliveryTypeName = /* @__PURE__ */ ((DeliveryTypeName3) => {
  DeliveryTypeName3["Morning"] = "morning";
  DeliveryTypeName3["Standard"] = "standard";
  DeliveryTypeName3["Evening"] = "evening";
  DeliveryTypeName3["Pickup"] = "pickup";
  DeliveryTypeName3["Express"] = "express";
  return DeliveryTypeName3;
})(DeliveryTypeName || {});
var DELIVERY_TYPE_NAMES_TO_IDS = {
  ["morning" /* Morning */]: 1 /* Morning */,
  ["standard" /* Standard */]: 2 /* Standard */,
  ["evening" /* Evening */]: 3 /* Evening */,
  ["pickup" /* Pickup */]: 4 /* Pickup */,
  ["express" /* Express */]: 7 /* Express */
};
var DELIVERY_TYPE_IDS_TO_NAMES = {
  [1 /* Morning */]: "morning" /* Morning */,
  [2 /* Standard */]: "standard" /* Standard */,
  [3 /* Evening */]: "evening" /* Evening */,
  [4 /* Pickup */]: "pickup" /* Pickup */,
  [7 /* Express */]: "express" /* Express */
};

// src/types/packageType.ts
var PackageTypeId = /* @__PURE__ */ ((PackageTypeId2) => {
  PackageTypeId2[PackageTypeId2["Package"] = 1] = "Package";
  PackageTypeId2[PackageTypeId2["Mailbox"] = 2] = "Mailbox";
  PackageTypeId2[PackageTypeId2["Letter"] = 3] = "Letter";
  PackageTypeId2[PackageTypeId2["DigitalStamp"] = 4] = "DigitalStamp";
  PackageTypeId2[PackageTypeId2["Pallet"] = 5] = "Pallet";
  PackageTypeId2[PackageTypeId2["PackageSmall"] = 6] = "PackageSmall";
  return PackageTypeId2;
})(PackageTypeId || {});
var PackageTypeName = /* @__PURE__ */ ((PackageTypeName2) => {
  PackageTypeName2["Package"] = "package";
  PackageTypeName2["Mailbox"] = "mailbox";
  PackageTypeName2["Letter"] = "letter";
  PackageTypeName2["DigitalStamp"] = "digital_stamp";
  PackageTypeName2["Pallet"] = "pallet";
  PackageTypeName2["PackageSmall"] = "package_small";
  return PackageTypeName2;
})(PackageTypeName || {});
var PACKAGE_TYPE_NAMES_TO_IDS = {
  ["package" /* Package */]: 1 /* Package */,
  ["mailbox" /* Mailbox */]: 2 /* Mailbox */,
  ["letter" /* Letter */]: 3 /* Letter */,
  ["digital_stamp" /* DigitalStamp */]: 4 /* DigitalStamp */,
  ["pallet" /* Pallet */]: 5 /* Pallet */,
  ["package_small" /* PackageSmall */]: 6 /* PackageSmall */
};
var PACKAGE_TYPE_IDS_TO_NAMES = {
  [1 /* Package */]: "package" /* Package */,
  [2 /* Mailbox */]: "mailbox" /* Mailbox */,
  [3 /* Letter */]: "letter" /* Letter */,
  [4 /* DigitalStamp */]: "digital_stamp" /* DigitalStamp */,
  [5 /* Pallet */]: "pallet" /* Pallet */,
  [6 /* PackageSmall */]: "package_small" /* PackageSmall */
};

// src/types/platform.ts
var PlatformId = /* @__PURE__ */ ((PlatformId3) => {
  PlatformId3[PlatformId3["MyParcel"] = 1] = "MyParcel";
  PlatformId3[PlatformId3["Flespakket"] = 2] = "Flespakket";
  PlatformId3[PlatformId3["SendMyParcel"] = 3] = "SendMyParcel";
  return PlatformId3;
})(PlatformId || {});
var PlatformName = /* @__PURE__ */ ((PlatformName3) => {
  PlatformName3["MyParcel"] = "myparcel";
  PlatformName3["Flespakket"] = "flespakket";
  PlatformName3["SendMyParcel"] = "belgie";
  return PlatformName3;
})(PlatformName || {});
var PlatformDomain = /* @__PURE__ */ ((PlatformDomain2) => {
  PlatformDomain2["MyParcel"] = "myparcel.nl";
  PlatformDomain2["Flespakket"] = "flespakket.nl";
  PlatformDomain2["SendMyParcel"] = "sendmyparcel.be";
  return PlatformDomain2;
})(PlatformDomain || {});
var PLATFORM_NAMES_TO_IDS = {
  ["myparcel" /* MyParcel */]: 1 /* MyParcel */,
  ["belgie" /* SendMyParcel */]: 3 /* SendMyParcel */,
  ["flespakket" /* Flespakket */]: 2 /* Flespakket */
};
var PLATFORM_IDS_TO_NAMES = {
  [1 /* MyParcel */]: "myparcel" /* MyParcel */,
  [3 /* SendMyParcel */]: "belgie" /* SendMyParcel */,
  [2 /* Flespakket */]: "flespakket" /* Flespakket */
};
var PLATFORM_IDS_TO_DOMAINS = {
  [1 /* MyParcel */]: "myparcel.nl" /* MyParcel */,
  [3 /* SendMyParcel */]: "sendmyparcel.be" /* SendMyParcel */,
  [2 /* Flespakket */]: "flespakket.nl" /* Flespakket */
};
var PLATFORM_NAMES_TO_DOMAINS = {
  ["myparcel" /* MyParcel */]: "myparcel.nl" /* MyParcel */,
  ["belgie" /* SendMyParcel */]: "sendmyparcel.be" /* SendMyParcel */,
  ["flespakket" /* Flespakket */]: "flespakket.nl" /* Flespakket */
};

// src/types/shipmentOptionName.ts
var ShipmentOptionName = /* @__PURE__ */ ((ShipmentOptionName2) => {
  ShipmentOptionName2["AgeCheck"] = "age_check";
  ShipmentOptionName2["Collect"] = "collect";
  ShipmentOptionName2["CooledDelivery"] = "cooled_delivery";
  ShipmentOptionName2["Insurance"] = "insurance";
  ShipmentOptionName2["LargeFormat"] = "large_format";
  ShipmentOptionName2["OnlyRecipient"] = "only_recipient";
  ShipmentOptionName2["PrinterlessReturn"] = "printerless_return";
  ShipmentOptionName2["Return"] = "return";
  ShipmentOptionName2["SameDayDelivery"] = "same_day_delivery";
  ShipmentOptionName2["Signature"] = "signature";
  return ShipmentOptionName2;
})(ShipmentOptionName || {});

// src/types/shipmentStatus.ts
var ShipmentStatus = /* @__PURE__ */ ((ShipmentStatus2) => {
  ShipmentStatus2[ShipmentStatus2["PendingConcept"] = 1] = "PendingConcept";
  ShipmentStatus2[ShipmentStatus2["PendingRegistered"] = 2] = "PendingRegistered";
  ShipmentStatus2[ShipmentStatus2["EnRouteHandedToCarrier"] = 3] = "EnRouteHandedToCarrier";
  ShipmentStatus2[ShipmentStatus2["EnRouteSorting"] = 4] = "EnRouteSorting";
  ShipmentStatus2[ShipmentStatus2["EnRouteDistribution"] = 5] = "EnRouteDistribution";
  ShipmentStatus2[ShipmentStatus2["EnRouteCustoms"] = 6] = "EnRouteCustoms";
  ShipmentStatus2[ShipmentStatus2["DeliveredAtRecipient"] = 7] = "DeliveredAtRecipient";
  ShipmentStatus2[ShipmentStatus2["DeliveredReadyForPickup"] = 8] = "DeliveredReadyForPickup";
  ShipmentStatus2[ShipmentStatus2["DeliveredPackagePickedUp"] = 9] = "DeliveredPackagePickedUp";
  ShipmentStatus2[ShipmentStatus2["DeliveredReturnShipmentReadyForPickup"] = 10] = "DeliveredReturnShipmentReadyForPickup";
  ShipmentStatus2[ShipmentStatus2["DeliveredReturnShipmentPackagePickedUp"] = 11] = "DeliveredReturnShipmentPackagePickedUp";
  ShipmentStatus2[ShipmentStatus2["PrintedLetter"] = 12] = "PrintedLetter";
  ShipmentStatus2[ShipmentStatus2["InactiveCredited"] = 13] = "InactiveCredited";
  ShipmentStatus2[ShipmentStatus2["PrintedDigitalStamp"] = 14] = "PrintedDigitalStamp";
  ShipmentStatus2[ShipmentStatus2["InactiveConcept"] = 30] = "InactiveConcept";
  ShipmentStatus2[ShipmentStatus2["InactiveRegistered"] = 31] = "InactiveRegistered";
  ShipmentStatus2[ShipmentStatus2["InactiveEnRouteHandedToCarrier"] = 32] = "InactiveEnRouteHandedToCarrier";
  ShipmentStatus2[ShipmentStatus2["InactiveEnRouteSorting"] = 33] = "InactiveEnRouteSorting";
  ShipmentStatus2[ShipmentStatus2["InactiveEnRouteDistribution"] = 34] = "InactiveEnRouteDistribution";
  ShipmentStatus2[ShipmentStatus2["InactiveEnRouteCustoms"] = 35] = "InactiveEnRouteCustoms";
  ShipmentStatus2[ShipmentStatus2["InactiveDeliveredAtRecipient"] = 36] = "InactiveDeliveredAtRecipient";
  ShipmentStatus2[ShipmentStatus2["InactiveDeliveredReadyForPickup"] = 37] = "InactiveDeliveredReadyForPickup";
  ShipmentStatus2[ShipmentStatus2["InactiveDeliveredPackagePickedUp"] = 38] = "InactiveDeliveredPackagePickedUp";
  return ShipmentStatus2;
})(ShipmentStatus || {});
var SHIPMENT_STATUS_PENDING = [1 /* PendingConcept */, 2 /* PendingRegistered */];
var SHIPMENT_STATUS_EN_ROUTE = [
  3 /* EnRouteHandedToCarrier */,
  4 /* EnRouteSorting */,
  5 /* EnRouteDistribution */,
  6 /* EnRouteCustoms */
];
var SHIPMENT_STATUS_DELIVERED = [
  7 /* DeliveredAtRecipient */,
  8 /* DeliveredReadyForPickup */,
  9 /* DeliveredPackagePickedUp */,
  10 /* DeliveredReturnShipmentReadyForPickup */,
  11 /* DeliveredReturnShipmentPackagePickedUp */
];
var SHIPMENT_STATUS_PRINTED = [12 /* PrintedLetter */, 14 /* PrintedDigitalStamp */];
var SHIPMENT_STATUS_INACTIVE = [
  13 /* InactiveCredited */,
  30 /* InactiveConcept */,
  31 /* InactiveRegistered */,
  32 /* InactiveEnRouteHandedToCarrier */,
  33 /* InactiveEnRouteSorting */,
  34 /* InactiveEnRouteDistribution */,
  35 /* InactiveEnRouteCustoms */,
  36 /* InactiveDeliveredAtRecipient */,
  37 /* InactiveDeliveredReadyForPickup */,
  38 /* InactiveDeliveredPackagePickedUp */
];

// src/helpers/convertEnumValue.ts
import { isEnumValue } from "@myparcel/ts-utils";
var convertEnumValue = (enumObject, value) => {
  return isEnumValue(value, enumObject) ? value : enumObject[value];
};

// src/helpers/carrier.ts
var toCarrierName = (carrier) => {
  return convertEnumValue(CARRIER_IDS_TO_NAMES, carrier);
};
var toCarrierId = (carrier) => {
  return convertEnumValue(CARRIER_NAMES_TO_IDS, carrier);
};

// src/helpers/deliveryType.ts
var toDeliveryTypeName = (carrier) => {
  return convertEnumValue(DELIVERY_TYPE_IDS_TO_NAMES, carrier);
};
var toDeliveryTypeId = (carrier) => {
  return convertEnumValue(DELIVERY_TYPE_NAMES_TO_IDS, carrier);
};

// src/helpers/packageType.ts
var toPackageTypeName = (carrier) => {
  return convertEnumValue(PACKAGE_TYPE_IDS_TO_NAMES, carrier);
};
var toPackageTypeId = (carrier) => {
  return convertEnumValue(PACKAGE_TYPE_NAMES_TO_IDS, carrier);
};

// src/helpers/platform.ts
var toPlatformName = (carrier) => {
  return convertEnumValue(PLATFORM_IDS_TO_NAMES, carrier);
};
var toPlatformId = (carrier) => {
  return convertEnumValue(PLATFORM_NAMES_TO_IDS, carrier);
};
export {
  CARRIER_IDS_TO_NAMES,
  CARRIER_NAMES_TO_IDS,
  CarrierId,
  CarrierName,
  CustomsDeclarationContents,
  DELIVERY_TYPE_IDS_TO_NAMES,
  DELIVERY_TYPE_NAMES_TO_IDS,
  DeliveryTypeId,
  DeliveryTypeName,
  PACKAGE_TYPE_IDS_TO_NAMES,
  PACKAGE_TYPE_NAMES_TO_IDS,
  PLATFORM_IDS_TO_DOMAINS,
  PLATFORM_IDS_TO_NAMES,
  PLATFORM_NAMES_TO_DOMAINS,
  PLATFORM_NAMES_TO_IDS,
  PackageTypeId,
  PackageTypeName,
  PlatformDomain,
  PlatformId,
  PlatformName,
  SHIPMENT_STATUS_DELIVERED,
  SHIPMENT_STATUS_EN_ROUTE,
  SHIPMENT_STATUS_INACTIVE,
  SHIPMENT_STATUS_PENDING,
  SHIPMENT_STATUS_PRINTED,
  ShipmentOptionName,
  ShipmentStatus,
  convertEnumValue,
  toCarrierId,
  toCarrierName,
  toDeliveryTypeId,
  toDeliveryTypeName,
  toPackageTypeId,
  toPackageTypeName,
  toPlatformId,
  toPlatformName
};
