// @myparcel/constants/dist/states/index.js
// src/states/canada.ts
var ALBERTA = "AB";
var BRITISH_COLUMBIA = "BC";
var MANITOBA = "MB";
var NEW_BRUNSWICK = "NB";
var NEWFOUNDLAND_AND_LABRADOR = "NL";
var NOVA_SCOTIA = "NS";
var NORTHWEST_TERRITORIES = "NT";
var NUNAVUT = "NU";
var ONTARIO = "ON";
var PRINCE_EDWARD_ISLAND = "PE";
var SASKATCHEWAN = "SK";
var QUEBEC = "QC";
var YUKON = "YT";
var STATES_CANADA = [
  ALBERTA,
  BRITISH_COLUMBIA,
  MANITOBA,
  NEW_BRUNSWICK,
  NEWFOUNDLAND_AND_LABRADOR,
  NOVA_SCOTIA,
  NORTHWEST_TERRITORIES,
  NUNAVUT,
  ONTARIO,
  PRINCE_EDWARD_ISLAND,
  SASKATCHEWAN,
  QUEBEC,
  YUKON
];

// src/states/unitedStates.ts
var ALABAMA = "AL";
var ALASKA = "AK";
var ARIZONA = "AZ";
var ARKANSAS = "AR";
var CALIFORNIA = "CA";
var COLORADO = "CO";
var CONNECTICUT = "CT";
var DELAWARE = "DE";
var FLORIDA = "FL";
var GEORGIA = "GA";
var HAWAII = "HI";
var IDAHO = "ID";
var ILLINOIS = "IL";
var INDIANA = "IN";
var IOWA = "IA";
var KANSAS = "KS";
var KENTUCKY = "KY";
var LOUISIANA = "LA";
var MAINE = "ME";
var MARYLAND = "MD";
var MASSACHUSETTS = "MA";
var MICHIGAN = "MI";
var MINNESOTA = "MN";
var MISSISSIPPI = "MS";
var MISSOURI = "MO";
var MONTANA = "MT";
var NEBRASKA = "NE";
var NEVADA = "NV";
var NEW_HAMPSHIRE = "NH";
var NEW_JERSEY = "NJ";
var NEW_MEXICO = "NM";
var NEW_YORK = "NY";
var NORTH_CAROLINA = "NC";
var NORTH_DAKOTA = "ND";
var OHIO = "OH";
var OKLAHOMA = "OK";
var OREGON = "OR";
var PENNSYLVANIA = "PA";
var RHODE_ISLAND = "RI";
var SOUTH_CAROLINA = "SC";
var SOUTH_DAKOTA = "SD";
var TENNESSEE = "TN";
var TEXAS = "TX";
var UTAH = "UT";
var VERMONT = "VT";
var VIRGINIA = "VA";
var WASHINGTON = "WA";
var WEST_VIRGINIA = "WV";
var WISCONSIN = "WI";
var WYOMING = "WY";
var STATES_US = [
  ALABAMA,
  ALASKA,
  ARIZONA,
  ARKANSAS,
  CALIFORNIA,
  COLORADO,
  CONNECTICUT,
  DELAWARE,
  FLORIDA,
  GEORGIA,
  HAWAII,
  IDAHO,
  ILLINOIS,
  INDIANA,
  IOWA,
  KANSAS,
  KENTUCKY,
  LOUISIANA,
  MAINE,
  MARYLAND,
  MASSACHUSETTS,
  MICHIGAN,
  MINNESOTA,
  MISSISSIPPI,
  MISSOURI,
  MONTANA,
  NEBRASKA,
  NEVADA,
  NEW_HAMPSHIRE,
  NEW_JERSEY,
  NEW_MEXICO,
  NEW_YORK,
  NORTH_CAROLINA,
  NORTH_DAKOTA,
  OHIO,
  OKLAHOMA,
  OREGON,
  PENNSYLVANIA,
  RHODE_ISLAND,
  SOUTH_CAROLINA,
  SOUTH_DAKOTA,
  TENNESSEE,
  TEXAS,
  UTAH,
  VERMONT,
  VIRGINIA,
  WASHINGTON,
  WEST_VIRGINIA,
  WISCONSIN,
  WYOMING
];
export {
  ALABAMA,
  ALASKA,
  ALBERTA,
  ARIZONA,
  ARKANSAS,
  BRITISH_COLUMBIA,
  CALIFORNIA,
  COLORADO,
  CONNECTICUT,
  DELAWARE,
  FLORIDA,
  GEORGIA,
  HAWAII,
  IDAHO,
  ILLINOIS,
  INDIANA,
  IOWA,
  KANSAS,
  KENTUCKY,
  LOUISIANA,
  MAINE,
  MANITOBA,
  MARYLAND,
  MASSACHUSETTS,
  MICHIGAN,
  MINNESOTA,
  MISSISSIPPI,
  MISSOURI,
  MONTANA,
  NEBRASKA,
  NEVADA,
  NEWFOUNDLAND_AND_LABRADOR,
  NEW_BRUNSWICK,
  NEW_HAMPSHIRE,
  NEW_JERSEY,
  NEW_MEXICO,
  NEW_YORK,
  NORTHWEST_TERRITORIES,
  NORTH_CAROLINA,
  NORTH_DAKOTA,
  NOVA_SCOTIA,
  NUNAVUT,
  OHIO,
  OKLAHOMA,
  ONTARIO,
  OREGON,
  PENNSYLVANIA,
  PRINCE_EDWARD_ISLAND,
  QUEBEC,
  RHODE_ISLAND,
  SASKATCHEWAN,
  SOUTH_CAROLINA,
  SOUTH_DAKOTA,
  STATES_CANADA,
  STATES_US,
  TENNESSEE,
  TEXAS,
  UTAH,
  VERMONT,
  VIRGINIA,
  WASHINGTON,
  WEST_VIRGINIA,
  WISCONSIN,
  WYOMING,
  YUKON
};
