// @myparcel/ts-utils/dist/index.cjs
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/index.ts
var src_exports = {};
__export(src_exports, {
  asyncEvery: () => asyncEvery,
  asyncSome: () => asyncSome,
  isEnumValue: () => isEnumValue,
  isInArray: () => isInArray,
  isOfType: () => isOfType,
  isUndefined: () => isUndefined,
  objectIsEqual: () => objectIsEqual,
  partitionArray: () => partitionArray,
  removePropertiesWith: () => removePropertiesWith,
  toArray: () => toArray
});
module.exports = __toCommonJS(src_exports);

// src/type-guards/isEnumValue.ts
function isEnumValue(key, enumObject) {
  return Object.values(enumObject).includes(key);
}

// src/type-guards/isInArray.ts
var isInArray = (value, array) => array.includes(value);

// src/type-guards/isOfType.ts
function isOfType(value, property) {
  return value?.[property] !== void 0;
}

// src/utils/asyncEvery.ts
var asyncEvery = async (arr, predicate) => {
  for (let i = 0; i < arr.length; i++) {
    if (!await predicate(arr[i], i, arr)) {
      return false;
    }
  }
  return true;
};

// src/utils/asyncSome.ts
var asyncSome = async (arr, predicate) => {
  for (let i = 0; i < arr.length; i++) {
    if (await predicate(arr[i], i, arr)) {
      return true;
    }
  }
  return false;
};

// src/utils/isUndefined.ts
var isUndefined = (value) => value === void 0;

// src/utils/objectIsEqual.ts
var objectIsEqual = (a, b) => JSON.stringify(a) === JSON.stringify(b);

// src/utils/partitionArray.ts
var partitionArray = (array, predicate) => {
  return (array ?? []).reduce(
    (acc, value) => {
      acc[predicate(value) ? 0 : 1].push(value);
      return acc;
    },
    [[], []]
  );
};

// src/utils/removePropertiesWith.ts
var removePropertiesWith = (object, predicate) => {
  return Object.entries(object).reduce((acc, [key, value]) => {
    if (!predicate(value)) {
      acc[key] = value;
    }
    return acc;
  }, {});
};

// src/utils/toArray.ts
var toArray = (itemOrItems, separator) => {
  if (itemOrItems === null || itemOrItems === void 0) {
    return [];
  }
  if (separator && typeof itemOrItems === "string") {
    return itemOrItems.split(separator);
  }
  return Array.isArray(itemOrItems) ? itemOrItems : [itemOrItems];
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  asyncEvery,
  asyncSome,
  isEnumValue,
  isInArray,
  isOfType,
  isUndefined,
  objectIsEqual,
  partitionArray,
  removePropertiesWith,
  toArray
});
