// @myparcel/constants/dist/countries/index.js
// src/countries/index.ts
var AFGHANISTAN = "AF";
var ALAND_ISLANDS = "AX";
var ALBANIA = "AL";
var ALGERIA = "DZ";
var AMERICAN_SAMOA = "AS";
var AMERICAN_VIRGIN_ISLANDS = "VI";
var ANDORRA = "AD";
var ANGOLA = "AO";
var ANGUILLA = "AI";
var ANTARCTICA = "AQ";
var ANTIGUA_AND_BARBUDA = "AG";
var ARGENTINA = "AR";
var ARMENIA = "AM";
var ARUBA = "AW";
var AUSTRALIA = "AU";
var AUSTRIA = "AT";
var AZERBAIJAN = "AZ";
var BAHAMAS = "BS";
var BAHRAIN = "BH";
var BANGLADESH = "BD";
var BARBADOS = "BB";
var BELARUS = "BY";
var BELGIUM = "BE";
var BELIZE = "BZ";
var BENIN = "BJ";
var BERMUDA = "BM";
var BHUTAN = "BT";
var BOLIVIA = "BO";
var BOSNIA_AND_HERZEGOVINA = "BA";
var BOTSWANA = "BW";
var BOUVET_ISLAND = "BV";
var BRAZIL = "BR";
var BRITISH_INDIAN_OCEAN_TERRITORY = "IO";
var BRITISH_VIRGIN_ISLANDS = "VG";
var BRUNEI_DARUSSALAM = "BN";
var BULGARIA = "BG";
var BURKINA_FASO = "BF";
var BURUNDI = "BI";
var CABO_VERDE = "CV";
var CAMBODIA = "KH";
var CAMEROON = "CM";
var CANADA = "CA";
var CAYMAN_ISLANDS = "KY";
var CENTRAL_AFRICAN_REPUBLIC = "CF";
var CHAD = "TD";
var CHILE = "CL";
var CHINA = "CN";
var CHRISTMAS_ISLAND = "CX";
var COCOS_ISLANDS = "CC";
var COLOMBIA = "CO";
var COMOROS = "KM";
var CONGO_BRAZZAVILLE = "CG";
var CONGO_KINSHASA = "CD";
var COOK_ISLANDS = "CK";
var COSTA_RICA = "CR";
var COTE_D_IVOIRE = "CI";
var CROATIA = "HR";
var CUBA = "CU";
var CURACAO = "CW";
var CYPRUS = "CY";
var CZECH_REPUBLIC = "CZ";
var DENMARK = "DK";
var DJIBOUTI = "DJ";
var DOMINICA = "DM";
var DOMINICAN_REPUBLIC = "DO";
var DUTCH_ANTILLES = "BQ";
var ECUADOR = "EC";
var EGYPT = "EG";
var EL_SALVADOR = "SV";
var EQUATORIAL_GUINEA = "GQ";
var ERITREA = "ER";
var ESTONIA = "EE";
var ESWATINI = "SZ";
var ETHIOPIA = "ET";
var FALKLAND_ISLANDS = "FK";
var FAROE_ISLANDS = "FO";
var FIJI = "FJ";
var FINLAND = "FI";
var FRANCE = "FR";
var FRENCH_GUIANA = "GF";
var FRENCH_POLYNESIA = "PF";
var FRENCH_SOUTHERN_TERRITORIES = "TF";
var GABON = "GA";
var GAMBIA = "GM";
var GEORGIA = "GE";
var GERMANY = "DE";
var GHANA = "GH";
var GIBRALTAR = "GI";
var GREECE = "GR";
var GREENLAND = "GL";
var GRENADA = "GD";
var GUADELOUPE = "GP";
var GUAM = "GU";
var GUATEMALA = "GT";
var GUERNSEY = "GG";
var GUINEA = "GN";
var GUINEA_BISSAU = "GW";
var GUYANA = "GY";
var HAITI = "HT";
var HEARD_ISLAND_AND_MCDONALD_ISLANDS = "HM";
var HOLY_SEE = "VA";
var HONDURAS = "HN";
var HONG_KONG = "HK";
var HUNGARY = "HU";
var ICELAND = "IS";
var INDIA = "IN";
var INDONESIA = "ID";
var IRAN = "IR";
var IRAQ = "IQ";
var IRELAND = "IE";
var ISLE_OF_MAN = "IM";
var ISRAEL = "IL";
var ITALY = "IT";
var JAMAICA = "JM";
var JAPAN = "JP";
var JERSEY = "JE";
var JORDAN = "JO";
var KAZAKHSTAN = "KZ";
var KENYA = "KE";
var KIRIBATI = "KI";
var KOREA = "KR";
var KOSOVO = "XK";
var KUWAIT = "KW";
var KYRGYZSTAN = "KG";
var LAO_PEOPLE_DEMOCRATIC_REPUBLIC = "LA";
var LATVIA = "LV";
var LEBANON = "LB";
var LESOTHO = "LS";
var LIBERIA = "LR";
var LIBYA = "LY";
var LIECHTENSTEIN = "LI";
var LITHUANIA = "LT";
var LUXEMBOURG = "LU";
var MACAO = "MO";
var MADAGASCAR = "MG";
var MALAWI = "MW";
var MALAYSIA = "MY";
var MALDIVES = "MV";
var MALI = "ML";
var MALTA = "MT";
var MARSHALL_ISLANDS = "MH";
var MARTINIQUE = "MQ";
var MAURITANIA = "MR";
var MAURITIUS = "MU";
var MAYOTTE = "YT";
var MEXICO = "MX";
var MICRONESIA = "FM";
var MOLDOVA = "MD";
var MONACO = "MC";
var MONGOLIA = "MN";
var MONTENEGRO = "ME";
var MONTSERRAT = "MS";
var MOROCCO = "MA";
var MOZAMBIQUE = "MZ";
var MYANMAR = "MM";
var NAMIBIA = "NA";
var NAURU = "NR";
var NEPAL = "NP";
var NETHERLANDS = "NL";
var NEW_CALEDONIA = "NC";
var NEW_ZEALAND = "NZ";
var NICARAGUA = "NI";
var NIGER = "NE";
var NIGERIA = "NG";
var NIUE = "NU";
var NORFOLK_ISLAND = "NF";
var NORTHERN_MARIANA_ISLANDS = "MP";
var NORTH_MACEDONIA = "MK";
var NORWAY = "NO";
var OMAN = "OM";
var PAKISTAN = "PK";
var PALAU = "PW";
var PALESTINE = "PS";
var PANAMA = "PA";
var PAPUA_NEW_GUINEA = "PG";
var PARAGUAY = "PY";
var PERU = "PE";
var PHILIPPINES = "PH";
var PITCAIRN = "PN";
var POLAND = "PL";
var PORTUGAL = "PT";
var PUERTO_RICO = "PR";
var QATAR = "QA";
var REUNION = "RE";
var ROMANIA = "RO";
var RUSSIAN_FEDERATION = "RU";
var RWANDA = "RW";
var SAINT_BARTHELEMY = "BL";
var SAINT_HELENA = "SH";
var SAINT_KITTS_AND_NEVIS = "KN";
var SAINT_LUCIA = "LC";
var SAINT_MARTIN = "MF";
var SAINT_PIERRE_AND_MIQUELON = "PM";
var SAINT_VINCENT_AND_THE_GRENADINES = "VC";
var SAMOA = "WS";
var SAN_MARINO = "SM";
var SAO_TOME_AND_PRINCIPE = "ST";
var SAUDI_ARABIA = "SA";
var SENEGAL = "SN";
var SERBIA = "RS";
var SEYCHELLES = "SC";
var SIERRA_LEONE = "SL";
var SINGAPORE = "SG";
var SINT_MAARTEN = "SX";
var SLOVAKIA = "SK";
var SLOVENIA = "SI";
var SOLOMON_ISLANDS = "SB";
var SOMALIA = "SO";
var SOUTH_AFRICA = "ZA";
var SOUTH_GEORGIA_AND_THE_SOUTH_SANDWICH_ISLANDS = "GS";
var SOUTH_SUDAN = "SS";
var SPAIN = "ES";
var SRI_LANKA = "LK";
var SUDAN = "SD";
var SURINAME = "SR";
var SVALBARD_AND_JAN_MAYEN = "SJ";
var SWEDEN = "SE";
var SWITZERLAND = "CH";
var SYRIAN_ARAB_REPUBLIC = "SY";
var TAIWAN = "TW";
var TAJIKISTAN = "TJ";
var TANZANIA = "TZ";
var THAILAND = "TH";
var TIMOR_LESTE = "TL";
var TOGO = "TG";
var TOKELAU = "TK";
var TONGA = "TO";
var TRINIDAD_AND_TOBAGO = "TT";
var TUNISIA = "TN";
var TURKEY = "TR";
var TURKMENISTAN = "TM";
var TURKS_AND_CAICOS_ISLANDS = "TC";
var TUVALU = "TV";
var UGANDA = "UG";
var UKRAINE = "UA";
var UNITED_ARAB_EMIRATES = "AE";
var UNITED_KINGDOM = "GB";
var UNITED_STATES_MINOR_OUTLYING_ISLANDS = "UM";
var UNITED_STATES_OF_AMERICA = "US";
var URUGUAY = "UY";
var UZBEKISTAN = "UZ";
var VANUATU = "VU";
var VENEZUELA = "VE";
var VIETNAM = "VN";
var WALLIS_AND_FUTUNA = "WF";
var WESTERN_SAHARA = "EH";
var YEMEN = "YE";
var ZAMBIA = "ZM";
var ZIMBABWE = "ZW";
var EU_COUNTRIES = [
  AUSTRIA,
  BELGIUM,
  BULGARIA,
  CROATIA,
  CYPRUS,
  CZECH_REPUBLIC,
  DENMARK,
  ESTONIA,
  FINLAND,
  FRANCE,
  GERMANY,
  GREECE,
  HUNGARY,
  IRELAND,
  ITALY,
  LATVIA,
  LITHUANIA,
  LUXEMBOURG,
  MALTA,
  NETHERLANDS,
  POLAND,
  PORTUGAL,
  ROMANIA,
  SLOVAKIA,
  SLOVENIA,
  SPAIN,
  SWEDEN
];
var ERS_COUNTRIES = [
  AUSTRIA,
  BELGIUM,
  CZECH_REPUBLIC,
  ESTONIA,
  FINLAND,
  FRANCE,
  GERMANY,
  GREECE,
  IRELAND,
  ITALY,
  LUXEMBOURG,
  NETHERLANDS,
  SLOVAKIA,
  SLOVENIA,
  SPAIN,
  UNITED_KINGDOM,
  MALTA,
  PORTUGAL,
  LATVIA,
  LITHUANIA,
  ROMANIA,
  CROATIA,
  HUNGARY,
  BULGARIA
];
var ALL_COUNTRIES = [
  AFGHANISTAN,
  ALAND_ISLANDS,
  ALBANIA,
  ALGERIA,
  AMERICAN_SAMOA,
  AMERICAN_VIRGIN_ISLANDS,
  ANDORRA,
  ANGOLA,
  ANGUILLA,
  ANTARCTICA,
  ANTIGUA_AND_BARBUDA,
  ARGENTINA,
  ARMENIA,
  ARUBA,
  AUSTRALIA,
  AUSTRIA,
  AZERBAIJAN,
  BAHAMAS,
  BAHRAIN,
  BANGLADESH,
  BARBADOS,
  BELARUS,
  BELGIUM,
  BELIZE,
  BENIN,
  BERMUDA,
  BHUTAN,
  BOLIVIA,
  BOSNIA_AND_HERZEGOVINA,
  BOTSWANA,
  BOUVET_ISLAND,
  BRAZIL,
  BRITISH_INDIAN_OCEAN_TERRITORY,
  BRITISH_VIRGIN_ISLANDS,
  BRUNEI_DARUSSALAM,
  BULGARIA,
  BURKINA_FASO,
  BURUNDI,
  CABO_VERDE,
  CAMBODIA,
  CAMEROON,
  CANADA,
  CAYMAN_ISLANDS,
  CENTRAL_AFRICAN_REPUBLIC,
  CHAD,
  CHILE,
  CHINA,
  CHRISTMAS_ISLAND,
  COCOS_ISLANDS,
  COLOMBIA,
  COMOROS,
  CONGO_BRAZZAVILLE,
  CONGO_KINSHASA,
  COOK_ISLANDS,
  COSTA_RICA,
  COTE_D_IVOIRE,
  CROATIA,
  CUBA,
  CURACAO,
  CYPRUS,
  CZECH_REPUBLIC,
  DENMARK,
  DJIBOUTI,
  DOMINICA,
  DOMINICAN_REPUBLIC,
  DUTCH_ANTILLES,
  ECUADOR,
  EGYPT,
  EL_SALVADOR,
  EQUATORIAL_GUINEA,
  ERITREA,
  ESTONIA,
  ESWATINI,
  ETHIOPIA,
  FALKLAND_ISLANDS,
  FAROE_ISLANDS,
  FIJI,
  FINLAND,
  FRANCE,
  FRENCH_GUIANA,
  FRENCH_POLYNESIA,
  FRENCH_SOUTHERN_TERRITORIES,
  GABON,
  GAMBIA,
  GEORGIA,
  GERMANY,
  GHANA,
  GIBRALTAR,
  GREECE,
  GREENLAND,
  GRENADA,
  GUADELOUPE,
  GUAM,
  GUATEMALA,
  GUERNSEY,
  GUINEA,
  GUINEA_BISSAU,
  GUYANA,
  HAITI,
  HEARD_ISLAND_AND_MCDONALD_ISLANDS,
  HOLY_SEE,
  HONDURAS,
  HONG_KONG,
  HUNGARY,
  ICELAND,
  INDIA,
  INDONESIA,
  IRAN,
  IRAQ,
  IRELAND,
  ISLE_OF_MAN,
  ISRAEL,
  ITALY,
  JAMAICA,
  JAPAN,
  JERSEY,
  JORDAN,
  KAZAKHSTAN,
  KENYA,
  KIRIBATI,
  KOREA,
  KOSOVO,
  KUWAIT,
  KYRGYZSTAN,
  LAO_PEOPLE_DEMOCRATIC_REPUBLIC,
  LATVIA,
  LEBANON,
  LESOTHO,
  LIBERIA,
  LIBYA,
  LIECHTENSTEIN,
  LITHUANIA,
  LUXEMBOURG,
  MACAO,
  MADAGASCAR,
  MALAWI,
  MALAYSIA,
  MALDIVES,
  MALI,
  MALTA,
  MARSHALL_ISLANDS,
  MARTINIQUE,
  MAURITANIA,
  MAURITIUS,
  MAYOTTE,
  MEXICO,
  MICRONESIA,
  MOLDOVA,
  MONACO,
  MONGOLIA,
  MONTENEGRO,
  MONTSERRAT,
  MOROCCO,
  MOZAMBIQUE,
  MYANMAR,
  NAMIBIA,
  NAURU,
  NEPAL,
  NETHERLANDS,
  NEW_CALEDONIA,
  NEW_ZEALAND,
  NICARAGUA,
  NIGER,
  NIGERIA,
  NIUE,
  NORFOLK_ISLAND,
  NORTHERN_MARIANA_ISLANDS,
  NORTH_MACEDONIA,
  NORWAY,
  OMAN,
  PAKISTAN,
  PALAU,
  PALESTINE,
  PANAMA,
  PAPUA_NEW_GUINEA,
  PARAGUAY,
  PERU,
  PHILIPPINES,
  PITCAIRN,
  POLAND,
  PORTUGAL,
  PUERTO_RICO,
  QATAR,
  REUNION,
  ROMANIA,
  RUSSIAN_FEDERATION,
  RWANDA,
  SAINT_BARTHELEMY,
  SAINT_HELENA,
  SAINT_KITTS_AND_NEVIS,
  SAINT_LUCIA,
  SAINT_MARTIN,
  SAINT_PIERRE_AND_MIQUELON,
  SAINT_VINCENT_AND_THE_GRENADINES,
  SAMOA,
  SAN_MARINO,
  SAO_TOME_AND_PRINCIPE,
  SAUDI_ARABIA,
  SENEGAL,
  SERBIA,
  SEYCHELLES,
  SIERRA_LEONE,
  SINGAPORE,
  SINT_MAARTEN,
  SLOVAKIA,
  SLOVENIA,
  SOLOMON_ISLANDS,
  SOMALIA,
  SOUTH_AFRICA,
  SOUTH_GEORGIA_AND_THE_SOUTH_SANDWICH_ISLANDS,
  SOUTH_SUDAN,
  SPAIN,
  SRI_LANKA,
  SUDAN,
  SURINAME,
  SVALBARD_AND_JAN_MAYEN,
  SWEDEN,
  SWITZERLAND,
  SYRIAN_ARAB_REPUBLIC,
  TAIWAN,
  TAJIKISTAN,
  TANZANIA,
  THAILAND,
  TIMOR_LESTE,
  TOGO,
  TOKELAU,
  TONGA,
  TRINIDAD_AND_TOBAGO,
  TUNISIA,
  TURKEY,
  TURKMENISTAN,
  TURKS_AND_CAICOS_ISLANDS,
  TUVALU,
  UGANDA,
  UKRAINE,
  UNITED_ARAB_EMIRATES,
  UNITED_KINGDOM,
  UNITED_STATES_MINOR_OUTLYING_ISLANDS,
  UNITED_STATES_OF_AMERICA,
  URUGUAY,
  UZBEKISTAN,
  VANUATU,
  VENEZUELA,
  VIETNAM,
  WALLIS_AND_FUTUNA,
  WESTERN_SAHARA,
  YEMEN,
  ZAMBIA,
  ZIMBABWE
];
export {
  AFGHANISTAN,
  ALAND_ISLANDS,
  ALBANIA,
  ALGERIA,
  ALL_COUNTRIES,
  AMERICAN_SAMOA,
  AMERICAN_VIRGIN_ISLANDS,
  ANDORRA,
  ANGOLA,
  ANGUILLA,
  ANTARCTICA,
  ANTIGUA_AND_BARBUDA,
  ARGENTINA,
  ARMENIA,
  ARUBA,
  AUSTRALIA,
  AUSTRIA,
  AZERBAIJAN,
  BAHAMAS,
  BAHRAIN,
  BANGLADESH,
  BARBADOS,
  BELARUS,
  BELGIUM,
  BELIZE,
  BENIN,
  BERMUDA,
  BHUTAN,
  BOLIVIA,
  BOSNIA_AND_HERZEGOVINA,
  BOTSWANA,
  BOUVET_ISLAND,
  BRAZIL,
  BRITISH_INDIAN_OCEAN_TERRITORY,
  BRITISH_VIRGIN_ISLANDS,
  BRUNEI_DARUSSALAM,
  BULGARIA,
  BURKINA_FASO,
  BURUNDI,
  CABO_VERDE,
  CAMBODIA,
  CAMEROON,
  CANADA,
  CAYMAN_ISLANDS,
  CENTRAL_AFRICAN_REPUBLIC,
  CHAD,
  CHILE,
  CHINA,
  CHRISTMAS_ISLAND,
  COCOS_ISLANDS,
  COLOMBIA,
  COMOROS,
  CONGO_BRAZZAVILLE,
  CONGO_KINSHASA,
  COOK_ISLANDS,
  COSTA_RICA,
  COTE_D_IVOIRE,
  CROATIA,
  CUBA,
  CURACAO,
  CYPRUS,
  CZECH_REPUBLIC,
  DENMARK,
  DJIBOUTI,
  DOMINICA,
  DOMINICAN_REPUBLIC,
  DUTCH_ANTILLES,
  ECUADOR,
  EGYPT,
  EL_SALVADOR,
  EQUATORIAL_GUINEA,
  ERITREA,
  ERS_COUNTRIES,
  ESTONIA,
  ESWATINI,
  ETHIOPIA,
  EU_COUNTRIES,
  FALKLAND_ISLANDS,
  FAROE_ISLANDS,
  FIJI,
  FINLAND,
  FRANCE,
  FRENCH_GUIANA,
  FRENCH_POLYNESIA,
  FRENCH_SOUTHERN_TERRITORIES,
  GABON,
  GAMBIA,
  GEORGIA,
  GERMANY,
  GHANA,
  GIBRALTAR,
  GREECE,
  GREENLAND,
  GRENADA,
  GUADELOUPE,
  GUAM,
  GUATEMALA,
  GUERNSEY,
  GUINEA,
  GUINEA_BISSAU,
  GUYANA,
  HAITI,
  HEARD_ISLAND_AND_MCDONALD_ISLANDS,
  HOLY_SEE,
  HONDURAS,
  HONG_KONG,
  HUNGARY,
  ICELAND,
  INDIA,
  INDONESIA,
  IRAN,
  IRAQ,
  IRELAND,
  ISLE_OF_MAN,
  ISRAEL,
  ITALY,
  JAMAICA,
  JAPAN,
  JERSEY,
  JORDAN,
  KAZAKHSTAN,
  KENYA,
  KIRIBATI,
  KOREA,
  KOSOVO,
  KUWAIT,
  KYRGYZSTAN,
  LAO_PEOPLE_DEMOCRATIC_REPUBLIC,
  LATVIA,
  LEBANON,
  LESOTHO,
  LIBERIA,
  LIBYA,
  LIECHTENSTEIN,
  LITHUANIA,
  LUXEMBOURG,
  MACAO,
  MADAGASCAR,
  MALAWI,
  MALAYSIA,
  MALDIVES,
  MALI,
  MALTA,
  MARSHALL_ISLANDS,
  MARTINIQUE,
  MAURITANIA,
  MAURITIUS,
  MAYOTTE,
  MEXICO,
  MICRONESIA,
  MOLDOVA,
  MONACO,
  MONGOLIA,
  MONTENEGRO,
  MONTSERRAT,
  MOROCCO,
  MOZAMBIQUE,
  MYANMAR,
  NAMIBIA,
  NAURU,
  NEPAL,
  NETHERLANDS,
  NEW_CALEDONIA,
  NEW_ZEALAND,
  NICARAGUA,
  NIGER,
  NIGERIA,
  NIUE,
  NORFOLK_ISLAND,
  NORTHERN_MARIANA_ISLANDS,
  NORTH_MACEDONIA,
  NORWAY,
  OMAN,
  PAKISTAN,
  PALAU,
  PALESTINE,
  PANAMA,
  PAPUA_NEW_GUINEA,
  PARAGUAY,
  PERU,
  PHILIPPINES,
  PITCAIRN,
  POLAND,
  PORTUGAL,
  PUERTO_RICO,
  QATAR,
  REUNION,
  ROMANIA,
  RUSSIAN_FEDERATION,
  RWANDA,
  SAINT_BARTHELEMY,
  SAINT_HELENA,
  SAINT_KITTS_AND_NEVIS,
  SAINT_LUCIA,
  SAINT_MARTIN,
  SAINT_PIERRE_AND_MIQUELON,
  SAINT_VINCENT_AND_THE_GRENADINES,
  SAMOA,
  SAN_MARINO,
  SAO_TOME_AND_PRINCIPE,
  SAUDI_ARABIA,
  SENEGAL,
  SERBIA,
  SEYCHELLES,
  SIERRA_LEONE,
  SINGAPORE,
  SINT_MAARTEN,
  SLOVAKIA,
  SLOVENIA,
  SOLOMON_ISLANDS,
  SOMALIA,
  SOUTH_AFRICA,
  SOUTH_GEORGIA_AND_THE_SOUTH_SANDWICH_ISLANDS,
  SOUTH_SUDAN,
  SPAIN,
  SRI_LANKA,
  SUDAN,
  SURINAME,
  SVALBARD_AND_JAN_MAYEN,
  SWEDEN,
  SWITZERLAND,
  SYRIAN_ARAB_REPUBLIC,
  TAIWAN,
  TAJIKISTAN,
  TANZANIA,
  THAILAND,
  TIMOR_LESTE,
  TOGO,
  TOKELAU,
  TONGA,
  TRINIDAD_AND_TOBAGO,
  TUNISIA,
  TURKEY,
  TURKMENISTAN,
  TURKS_AND_CAICOS_ISLANDS,
  TUVALU,
  UGANDA,
  UKRAINE,
  UNITED_ARAB_EMIRATES,
  UNITED_KINGDOM,
  UNITED_STATES_MINOR_OUTLYING_ISLANDS,
  UNITED_STATES_OF_AMERICA,
  URUGUAY,
  UZBEKISTAN,
  VANUATU,
  VENEZUELA,
  VIETNAM,
  WALLIS_AND_FUTUNA,
  WESTERN_SAHARA,
  YEMEN,
  ZAMBIA,
  ZIMBABWE
};
